app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,53 @@
+class RestaurantsController < ApplicationController
+
+  def index
+    @restaurants = Restaurant.all.order("name ASC")
```

Comment:
paginate here. If you go viral and get 5000 restaurants, you'll run into a performance. Use kaminari to paginate this data or limit the result set.